workflow-service/src/test/java/com/redhat/parodos/workflow/execution/service/WorkFlowServiceImplTest.java

```diff
@@ -907,6 +909,27 @@ public void testGetWorkflowParametersWithoutWorkflowOptions() {
 		assertNull(workflowParameters.getWorkFlowOptions().getNewOptions());
 	}
 
+	@Test
+	void getWorkFlowStatusByProjectId_when_projectIsFound_then_returnWorkFlowStatus() {
```

Comment:
We need to increase testing when no workflows with valid projects and what happens when JPA repo returns nil